clinica/iotools/bids_utils.py

```diff
@@ -423,62 +448,68 @@ def create_scans_dict(
 
-    Args:
-        study_name: name of the study (Ex ADNI)
-        bids_dir: path to the bids directory
-    """
-    import json
-    from os import path
+    if bids_version:
+        bids_desc = BIDSDatasetDescription(name=study_name, bids_version=bids_version)
+    else:
+        bids_desc = BIDSDatasetDescription(name=study_name)
+    with open(Path(bids_dir) / "dataset_description.json", "w") as f:
+        bids_desc.write(to=f)
 
-    import clinica
-    from clinica.iotools.bids_dataset_description import BIDSDatasetDescription
 
-    with open(path.join(bids_dir, "dataset_description.json"), "w") as f:
-        BIDSDatasetDescription(name=study_name).write(to=f)
+def _write_readme(bids_dir: Union[str, Path]) -> None:
+    """Write `README`file at the root of the BIDS directory."""
+    import clinica
 
-    with open(path.join(bids_dir, "README"), "w") as f:
+    with open(Path(bids_dir) / "README", "w") as f:
         f.write(
             f"This BIDS directory was generated with Clinica v{clinica.__version__}.\n"
             f"More information on http://www.clinica.run\n"
         )
 
-    validator_dict = {
-        "ignore": [
-            # Possibly dcm2nii(x) errors
-            "NIFTI_UNIT",
-            "INCONSISTENT_PARAMETERS",
-            # fMRI-specific errors
-            "SLICE_TIMING_NOT_DEFINED",
-            "NIFTI_PIXDIM4",
-            "BOLD_NOT_4D",
-            "REPETITION_TIME_MUST_DEFINE",
-            "TASK_NAME_MUST_DEFINE",
-            # Won't fix errors
-            "MISSING_SESSION",  # Allows subjects to have different sessions
-            "INCONSISTENT_SUBJECTS",  # Allows subjects to have different modalities
-            "SCANS_FILENAME_NOT_MATCH_DATASET",  # Necessary until PET is added to BIDS standard
-            "CUSTOM_COLUMN_WITHOUT_DESCRIPTION",  # We won't create these JSON files as clinical description
-            # is already done in TSV files of clinica.
-            "NO_AUTHORS",  # Optional field in dataset_description.json
-        ],
-        "warn": [],
-        "error": [],
-        "ignoredFiles": [],
-    }
 
-    with open(path.join(bids_dir, ".bids-validator-config.json"), "w") as f:
-        json.dump(validator_dict, f, skipkeys=True, indent=4)
+def _write_bids_validator_config(bids_dir: Union[str, Path]) -> None:
+    """Write `.bids-validator-config.json` at the root of the BIDS directory."""
+    import json
+
+    with open(Path(bids_dir) / ".bids-validator-config.json", "w") as f:
+        json.dump(BIDS_VALIDATOR_CONFIG, f, skipkeys=True, indent=4)
+
 
-    with open(path.join(bids_dir, ".bidsignore"), "w") as f:
+def _write_bidsignore(bids_dir: Union[str, Path]) -> None:
+    """Write `.bidsignore` file at the root of the BIDS directory."""
+    with open(Path(bids_dir) / ".bidsignore", "w") as f:
         # pet/ is necessary until PET is added to BIDS standard
         f.write("\n".join(["pet/", "conversion_info/"]))
```

Comment:
We are now compliant with BIDS for PET.

```suggestion
        f.write("\n".join(["conversion_info/"]))
```